src/FakeItEasy.Analyzer/UnusedReturnValueAnalyzer.cs

```diff
@@ -42,13 +39,8 @@ private static void AnalyzeCall(SyntaxNodeAnalysisContext context)
                 return;
             }
 
-            var diagnosticName = GetDiagnosticName(methodSymbol);
-            if (diagnosticName == null)
-            {
-                return;
-            }
-
-            var descriptor = Diagnostics.GetValueOrDefault(diagnosticName);
+            string methodFullName = $"{methodSymbol.ContainingType.GetFullName()}.{methodSymbol.GetDecoratedName()}";

```

Comment:
Anyway, I feel bad about the nitpick now :worried: - happy to merge when I get the button.